L1Trigger/L1THGCal/plugins/ntuples/HGCalTriggerNtupleHGCDigis.cc

```diff
@@ -173,11 +218,33 @@ fill(const edm::Event& e, const edm::EventSetup& es)
           hgcdigi_simenergy_.emplace_back(hit_energy); 
         }
       }
+
+     for(const auto& digi : bh_digis)
+      {
+        const HcalDetId id(digi.id());
+        bhdigi_id_.emplace_back(id.rawId());
+        bhdigi_subdet_.emplace_back(id.subdetId());
+        bhdigi_side_.emplace_back(id.zside());
+        bhdigi_layer_.emplace_back(id.depth());
+        bhdigi_ieta_.emplace_back(id.ieta());
+        bhdigi_iphi_.emplace_back(id.iphi());
+        GlobalPoint cellpos = triggerGeometry_->bhGeometry().getPosition(id.rawId());
+        bhdigi_eta_.emplace_back(cellpos.eta());
+        bhdigi_phi_.emplace_back(cellpos.phi());
+        bhdigi_z_.emplace_back(cellpos.z());
+        bhdigi_data_.emplace_back(digi[2].data()); 
```

Comment:
Yes, this is the t=0 (in-time bx) sample. It seems indeed not possible to retrieve this value from some parameter; actually even in the HGCAL digitization it is hardcoded everywhere. What I can do right now is to define a local constant variable, which would at least better than using `2`.